### PR TITLE
feat(odiff): replace pixelmatch

### PR DESCRIPTION
**Pull request by JonasBa**
First PR that replace pixelmatch with odiff. Note that this should not be merged because it will fail to compile (vercel/ncc does not handle binary paths that are copied over correctly).

I will open a followup stacked PR with only changes to the build step so that they can be atomically reviewed.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -19,25 +19,24 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: 12
-    - name: yarn install
+    - name: npm install
```
Comment (billyvg) [inline .github/workflows/release.yml:22]: Why change this to npm?

File: src/util/fileToPng.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import {PNG} from 'pngjs';
 /**
  * Given a path to a PNG, returns a pngjs object
  */
+// eslint-disable-next-line
```
Comment (billyvg) [inline src/util/fileToPng.ts:7]: Is this needed? Can we add the specific rule to disable?

Comment (JonasBa) [reply src/util/fileToPng.ts:7]: I will just remove this file entirely, I think it's unused

File: .github/workflows/release.yml (hunk 1 above)
Comment (JonasBa) [reply .github/workflows/release.yml]: @billyvg I reverted it, wasnt necessary in the end

**Comment by JonasBa** [comment]
This now contains all of the stacked PR's that were merged here (I wish gh was better with stacked PR's), but I'll make sure to remove dist folder entirely so we can give this another look before merging without looking at the compiled files along with it.

**Comment by JonasBa** [comment]
@billyvg this should be good to go now. The PR contains all the changes from the stacked PR's, we've been running it on getsentry/sentry for a week now and it's had some pretty good [results](https://sentry.io/organizations/sentry/discover/results/?display=top5&field=diffing_library&field=count%28%29&field=failure_rate%28%29&field=avg%28transaction.duration%29&field=p50%28transaction.duration%29&field=p95%28transaction.duration%29&name=All+Events&project=5324467&query=title%3Arun&sort=-diffing_library&statsPeriod=24h&yAxis=p95%28transaction.duration%29). I'll probably write a message to eng team, but I think we are much closer to the actual test flake percentage

**Review by billyvg — APPROVED** [review]

**Pull request merged**